Source/WebKit/Configurations/WebKit.xcconfig

```diff
@@ -238,13 +238,13 @@ WK_EXCLUDED_WEBPUSHD_SANDBOX_YES = com.apple.WebKit.webpushd.mac.sb
 
 EXCLUDED_EXTENSION_SOURCE_FILE_NAMES = Shared/AuxiliaryProcessExtensions/*;
 
-INCLUDED_EXTENSION_SOURCE_FILE_NAMES_YES[sdk=iphone*] = Shared/AuxiliaryProcessExtensions/*;
-INCLUDED_EXTENSION_SOURCE_FILE_NAMES_YES[sdk=iphone*17.0.internal] = ;
-INCLUDED_EXTENSION_SOURCE_FILE_NAMES_YES[sdk=iphone*17.1.internal] = ;
-INCLUDED_EXTENSION_SOURCE_FILE_NAMES_YES[sdk=iphone*17.2.internal] = ;
-INCLUDED_EXTENSION_SOURCE_FILE_NAMES_YES[sdk=iphone*17.3.internal] = ;
-INCLUDED_EXTENSION_SOURCE_FILE_NAMES_YES[sdk=iphone*18.0.internal] = ;
-INCLUDED_EXTENSION_SOURCE_FILE_NAMES_YES[sdk=xr*] = ;
+INCLUDED_EXTENSION_SOURCE_FILE_NAMES[sdk=iphone*] = Shared/AuxiliaryProcessExtensions/*;
+INCLUDED_EXTENSION_SOURCE_FILE_NAMES[sdk=iphone*17.0.internal] = ;
+INCLUDED_EXTENSION_SOURCE_FILE_NAMES[sdk=iphone*17.1.internal] = ;
+INCLUDED_EXTENSION_SOURCE_FILE_NAMES[sdk=iphone*17.2.internal] = ;
+INCLUDED_EXTENSION_SOURCE_FILE_NAMES[sdk=iphone*17.3.internal] = ;
+INCLUDED_EXTENSION_SOURCE_FILE_NAMES[sdk=iphone*18.0.internal] = ;
+INCLUDED_EXTENSION_SOURCE_FILE_NAMES[sdk=xr*] = ;
```

Comment:
Not caused by this patch, but why do we need to specify xr* but not watch* and appletv*?

Reply:
> Not caused by this patch, but why do we need to specify xr* but not watch* and appletv*?

Contextual information indicates it's because watchOS and tvOS don't have web browsers, and therefore weren't affected by the BrowserEngineKit changes in iOS 17.4.

Reply:
Also, it is normally OK to build these files, since they are guarded on the feature flag.

Thanks for reviewing, all!